src/common/task.cc

```diff
@@ -324,7 +324,10 @@ int TaskSpec_arg_id_count(TaskSpec *spec, int64_t arg_index) {
   RAY_CHECK(spec);
   auto message = flatbuffers::GetRoot<TaskInfo>(spec);
   auto ids = message->args()->Get(arg_index)->object_ids();
-  return ids->size();
+  if (ids == nullptr)
+    return 0;
+  else
+    return ids->size();
```

Comment:
The linting doesn't seem to be enforcing this, but please change this to

```c++
if (ids == nullptr) {
  return 0;
} else {
  return ids->size();
}
```

Reply:
Got it, thanks for your time, i will change them soon